content/references/rippled-api/public-rippled-methods/ledger-methods/ledger_entry.md

```diff
@@ -46,25 +46,33 @@ This method can retrieve several different types of data. You can select which t
 
 1. `index` - Retrieve any type of ledger object by its unique ID
 2. `account_root` - Retrieve an [AccountRoot object](accountroot.html). This is roughly equivalent to the [account_info method][].
-3. `directory` - Retrieve a [DirectoryNode](directorynode.html), which contains a list of other ledger objects
-4. `offer` - Retrieve an [Offer object](offer.html), which defines an offer to exchange currency
+3. `directory` - Retrieve a [DirectoryNode](directorynode.html), which contains a list of other ledger objects.
+4. `offer` - Retrieve an [Offer object](offer.html), which defines an offer to exchange currency.
 5. `ripple_state` - Retrieve a [RippleState object](ripplestate.html), which tracks a (non-XRP) currency balance between two accounts.
+6. `check` - Retrieve a [Check object](check.html), which is a potential payment that can be cashed by its recipient. [New in: rippled 1.0.0][]
+7. `escrow` - Retrieve an [Escrow object](escrow-object.html), which holds XRP until a specific time or condition is met. [New in: rippled 1.0.0][]
+8. `payment_channel` - Retireve a [PayChannel object](paychannel.html), which holds XRP for asynchronous payments. [New in: rippled 1.0.0][]
 
 If you specify more than one of the above items, the server retrieves only of them; it is undefined which it chooses.
```

Comment:
Typo?  "only _one_ of them;"?

Reply:
Nice! I think you just caught a typo that's been there at least 3 years. >_<